### PR TITLE
FEATURE: F1-51 Use bulk api for deleting users and groups

### PR DESCRIPTION
**Pull request by kandl**
- Use bulk api for deleting users and groups
- Enable filtering of the users / groups by data source
- Previous PR cleanup remnants.

JIRA: F1-51

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
// Tiger platform
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

```
// Bear platform
extended test - cypress - integrated
extended test - cypress - isolated
extended test - cypress - record
```

### CODE REVIEW

File: libs/sdk-ui-kit/src/Icon/icons/UserGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+        <svg
+            className={className}
+            width={width ?? 27}
+            height={height ?? 27}
+            viewBox="0 0 27 27"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <rect width="27" height="27" rx="13.5" fill="#EBEFF4" />
```
Comment (scavnickyj) [inline libs/sdk-ui-kit/src/Icon/icons/UserGroup.tsx:19]: I think we want another color variable for the fill here. Check other icons, I believe some of them have multiple colors.

File: libs/sdk-ui-kit/src/Icon/icons/User.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+        <svg
+            className={className}
+            width={width ?? 27}
+            height={height ?? 27}
+            viewBox="0 0 27 27"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <rect width="27" height="27" rx="13.5" fill="#EBEFF4" />
```
Comment (scavnickyj) [inline libs/sdk-ui-kit/src/Icon/icons/User.tsx:19]: Same here

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by scavnickyj — APPROVED** [review]

**Comment by scavnickyj** [comment]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4674

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4674

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**